examples/corb2-project/build/ml-last-configured-timestamps.properties

```diff
@@ -0,0 +1,4 @@
+#
```

Comment:
Can you add this to a .gitignore file in examples/corb2-project? Or better yet - there should be a way to ignore any "build" directory from the root .gitignore directory.